### DISCUSSION TITLE
Idea: In X-Plat Template define the default startup project

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#12082] [Ideas]
**Describe the bug**

**To Reproduce**

1. Make a project using any avalonia template
2. Click run

**Expected behavior**
Program should run just fine

**Screenshots**
![image](https://github.com/AvaloniaUI/Avalonia/assets/80383169/38fefe60-e566-4fa5-855f-8ac3850b8a9e)

**Desktop (please complete the following information):**
```
Edition	Windows 10 Pro
Version	22H2
Installed on	‎7/‎6/‎2023
OS build	19045.2006
Experience	Windows Feature Experience Pack 120.2212.4180.0

dotnet 7 latest visual studio install
```

**Additional context**
None

**Comment by ghost** [comment]
it seems if i make Avalonia.Desktop my startup project it will work fine, perhaps this should be the default when clicking run?

**Comment by timunie** [comment]
> it seems if i make Avalonia.Desktop my startup project it will work fine, perhaps this should be the default when clicking run?

Don't know if that is possible at all. Afaik it's controlled by your IDE. For example I guess VS always uses the first project found, unless it was changed by the user settings. @Takoooooo any idea on this ?

For now, let me move this thread under ideas section as it is not an Avalonia bug.

**Comment by studentsaj** [comment]
Hello,

New here & to Avalonia, thought I'd post my experience:

1. Fresh Install and same issue as reported.
3. Created a project/solution via the IDE, selected .Desktop as startup project, build and run caused a blue screen each time. Although running directly from the bin/Debug directory is fine. Third attempt+ seems OK - no crash. Although I did change the Target Framework from 7.0 to 8.0. 
4. Creating a project via the command line builds and runs fine.

**Reply by maxkatz6** [reply]
Blue Screen? That's something new.
